softdev.bib

```diff
@@ -1591,3 +1591,11 @@ @Article{killick12optimal
     Url     = {http://dx.doi.org/10.1080/01621459.2012.737745},
     month   = Dec,
 }
+
+@inproceedings{kalibera05precision,
+    title = "Benchmark precision and random initial state",
+    author = "Kalibera, Tomas and Bulej, Lubomir and Tuma, Petr",
+    booktitle = "SPECTS",
```

Comment:
I think the `booktitle` key is left unfiddled with, so it's OK (at least, we have many others in this format that do the right thing -- I told you the rules were a nightmare :/).